### ISSUE TITLE
osx(sierra): sample multiplication make black flash in editor , and edit cursor stay 

### ISSUE THREAD

**Issue by onukore** [kmatheussen/radium#867]
![capture d ecran 2017-03-10 a 10 12 23](https://cloud.githubusercontent.com/assets/13727495/23789211/1cd71c6e-057a-11e7-93c0-3731fdb68ae3.png)

**Comment by kmatheussen** [comment]
Hi,

What is sample multiplication? Do you mean changing any
number in the note duplicator?

**Comment by onukore** [comment]
hi 
 yes ,duplicator (always forget the name ;) )
yes when I change any value , black flash in editor happen and edit cursor stay like in picture

**Comment by onukore** [comment]
2 finger with track pad (scroll wheel) don't work in editor, but work with a mouse

**Comment by onukore** [comment]
other think
is possible to hide finder barre on radium like in renoise, because when you go with mouse in menu it always appear this is very arrrrrrggggggggghhhhhhhhh !!!!  fu........! in renoise that never happen 
![capture d ecran 2017-03-10 a 10 26 05](https://cloud.githubusercontent.com/assets/13727495/23789824/c20c46da-057c-11e7-83a8-049b18746cb3.png)
![capture d ecran 2017-03-10 a 10 26 52](https://cloud.githubusercontent.com/assets/13727495/23789829/c5910002-057c-11e7-8fa9-9532514fec1c.png)

**Comment by kmatheussen** [comment]
> yes when I change any value with mouse scroll wheel , black flash in
editor happen and edit cursor stay like in picture

From the picture, it seems like there are several text editor cursors that
are left in the note duplicator.
Is this only a visual thing? Or does something stop working too?

**Comment by kmatheussen** [comment]
> is possible to hide finder barre on radium like in renoise

What is "finder barre"?

**Comment by kmatheussen** [comment]
> What is "finder barre"?

Does the OSX system menu bar pop up in full screen mode when you access the
menues?

**Comment by onukore** [comment]
yes the OS X bar menu pop up in full screen, this is very arrrrrrrrggggh, in renoise menu bar is blocked never appear , 
for note duplicator, black flash happen in editor , and changing value is very slow and in start part value jump 10 per 10 not 1 by 1 with scroll wheel , look like very buggy

**Comment by kmatheussen** [comment]
I can't reproduce the black flashing and the cursor in the note duplicator thing, but perhaps you can download latest macos update (10.12.4) and see if it fixes any of the sierra issues?

**Comment by onukore** [comment]
hi ,
ok i will do that
